platforms/python/versions/bullseye/versionsToBuild.txt

```diff
@@ -1,24 +1,24 @@
+
+# NOTE: Make sure to set the default version in 'defaultVersion.txt' file
 # version, gpg keys, dockerfile
-3.7.9, 0D96DF4D4110E5C43FBFB17F2D347EA6AA65421D,
+3.10.0, A035C8C19219BA821ECEA86B64E628F8D684696D,
+3.10.0a2, A035C8C19219BA821ECEA86B64E628F8D684696D,
+3.10.12, A035C8C19219BA821ECEA86B64E628F8D684696D,
+3.10.4, A035C8C19219BA821ECEA86B64E628F8D684696D,
+3.10.8, A035C8C19219BA821ECEA86B64E628F8D684696D,
+3.11.0, A035C8C19219BA821ECEA86B64E628F8D684696D
+3.11.0b1, A035C8C19219BA821ECEA86B64E628F8D684696D,
+3.11.4, A035C8C19219BA821ECEA86B64E628F8D684696D,
```

Comment:
Nit: looks like the order of versions in this file (and other files) became somewhat random. By any chance, would you know where this is coming from?